.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
I don't think you intented to add `.idea` files to the PR.
You can look at https://stackoverflow.com/a/1753078 to add `.idea/` to your local git repo so it doesn't show up in unstaged files.